app/resolve/daemon/__init__.py

```diff
@@ -1,2 +1,26 @@
+import requests
+from os import environ
+import time
+import logging
+
+import core.config as config
+from core.execute_query import execute_query
+from core.parser import parser
+from core.rquest_dto.result import RquestResult
+
+
 def main() -> None:
-    print("hello rabbit world!")
+    logger = logging.getLogger(config.LOGGER_NAME)
+    result = execute_query(parser)
+    if not isinstance(result, RquestResult):
+        raise TypeError("Payload does not match RQuest result schema")
+    destination = environ.get("RESOLVE_DAEMON_POST_URL")
```

Comment:
I believe a `settings.py` is in #72 - but worth adding it here as this is ready. 
So a new `settings.py` in `core`, get the environment variable in there, and import that in this file.

Let's name this environment variable better too, will follow up with a comment.

Reply:
`TASK_API_BASE_URL` would work - and then this code can append the path.